theseus_gui/src/pages/Browse.vue

```diff
@@ -712,12 +720,12 @@ onUnmounted(() => unlistenOffline())
         @switch-page="onSearchChange"
       />
       <SplashScreen v-if="loading" />
-      <section v-else-if="offline && results.total_hits == 0" class="offline">
+      <section v-else-if="offline && results.total_hits === 0" class="offline">
         You are currently offline. Connect to the internet to browse Modrinth!
       </section>
       <section v-else class="project-list display-mode--list instance-results" role="list">
         <SearchCard
-          v-for="result in results.hits"
+          v-for="result in results.hits.filter((x) => !x.installed || !hideAlreadyInstalled)"
```

Comment:
not how we should do this, this should use filters in the search route call to filter out already installed projects (so the number of hits/results is the same)